clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -1175,8 +1175,9 @@ private static void printStorageNodeList() {
     printObject(nodeResponse);
   }
 
-  private static void printInstancesStatuses() {
-    MultiNodesStatusResponse nodeResponse = controllerClient.listInstancesStatuses();
+  private static void printInstancesStatuses(CommandLine cmd) {
+    String enableReplicas = getOptionalArgument(cmd, Arg.ENABLE_DISABLED_REPLICA, "false");
```

Comment:
If `ENABLE_DISABLED_REPLICA` is only assigned to `true` or `false` values, wondering if it makes more sense to use the non-parameterized `Arg` and its associated `hasOption` method to retrieve the boolean value. One of such example is `Arg.DISABLE_META_STORE`.

Reply:
Thats good suggestion. I will add it later as I just merged the PR

Reply:
Sure, thanks @majisourav99 !